Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -316,6 +317,10 @@ public static int CalculateClimbingChance(PlayerEntity player, int basePercentSu
         // Calculate how many uses a skill needs before its value will rise.
         public static int CalculateSkillUsesForAdvancement(int skillValue, int skillAdvancementMultiplier, float careerAdvancementMultiplier, int level)
         {
+            Func<int, int, float, int, int> del;
```

Comment:
In my opinion, this formula would be more useful to mods if it just passed the `DFCareer.Skills` enum. 

In PlayerEntity.cs, the code does
```
int skillAdvancementMultiplier = DaggerfallSkills.GetAdvancementMultiplier((DFCareer.Skills)i);
                float careerAdvancementMultiplier = Career.AdvancementMultiplier;
                int usesNeededForAdvancement = FormulaHelper.CalculateSkillUsesForAdvancement(skills.GetPermanentSkillValue(i), skillAdvancementMultiplier, careerAdvancementMultiplier, level);
```

The permanent skill value, the skill advancement multiplier, and the career advancement multiplier, and the current level can all be acquired from a reference to the player entity, which can be found in the GameManager.

Suggested fix: remove the lines from `PlayerEntity.RaiseSkills`, put them inside the default formula implementation, change the signature to only take a `DFCareer.Skills` enum, and adjust the override in consequence

Reply:
Going to leave this one as it is, I don't see much value in this change and I don't want to mess with this formula without good reason. It's been as it is for a long time and working as classic (we think)